### DISCUSSION TITLE
-kb-row-select seems to malfunction in combination with -sort

### DISCUSSION THREAD

**Discussion by Kzer-Za** [davatorium/rofi#1710] [(Possible) Issues] [answered]
I'm not sure if it's a bug, but this behavior seems strange:

Run:
```
rofi -show drun -no-config -kb-row-select "Tab" -kb-element-next "" -kb-screenshot "Print" -sort
```
(`-no-config` is there to make sure that my config is not interacting with the result.)

Start typing something that fits several items, select the one of them, press "Tab".

![rofi-2022-09-21-1525-00000](https://user-images.githubusercontent.com/26425206/191504329-13fbafc4-f958-4c9d-8f30-72015a091615.png)

As the result the selected item gets placed in the input text field, as it should. But also the cursor is now on a completely different item, so if you press Enter, that newly selected item is executed.

![rofi-2022-09-21-1525-00001](https://user-images.githubusercontent.com/26425206/191504353-3342fa0f-d3eb-4c36-a51f-77b171f0d877.png)

**Comment by DaveDavenport** [comment, marked as answer]
correct.. the position is of the selection is kept (and if that is no longer valid it takes the last one). Not the selection it self.

I wanted to change this, but given the backlash on each minor change, not going to change this.

**Reply by Kzer-Za** [reply]
Ok, got it.

However, could you in the future add an option for keeping the selected item instead of the position? If by default this option was false, then the people who want the old behavior would have nothing to complain about. If it's not to much trouble, of course. I totally understand that there is a lot of options already and you probably don't want to complicate the codebase further. Even with the current behavior rofi is the best launcher that I've tried!
